testsuite/oauth/authorize.py

```diff
@@ -156,7 +158,12 @@ def _execute_step(self, step):
         if not elem.is_displayed() and step.get('optional'):
             return
         elif not elem.is_displayed():
-            raise ElementNotFoundException('Element is hidden', self.browser)
+            try:
+                wait = WebDriverWait(self.browser, VISIBILITY_TIMEOUT)
```

Comment:
Allscripts was failing as if I hadn't clicked the big "sign in" link. Changing the viewport size and adding this wait seemed to help. The old viewport size looked like a mobile view, that may have something to do with it. Another commit is coming that reworks this a little bit, so that it'll wait for optional elements too.

![image](https://cloud.githubusercontent.com/assets/502000/17059696/6d8fe06e-4ff4-11e6-8682-8591e34992e3.png)